index.html

```diff
@@ -4,76 +4,453 @@
-            /></a>
-          </li>
-          <li class="social">
-            <a href="#"><img src="images/medium.svg" alt="medium-icon" /></a>
-          </li>
-          <li class="social">
-            <a href="#"><img src="images/github.svg" alt="github-icon" /></a>
-          </li>
-          <li class="social">
-            <a href="#"
-              ><img src="images/angellist.svg" alt="angellist-icon"
-            /></a>
-          </li>
-        </ul>
+      <!-- Social links -->
+      <ul class="social-media-icons">
+        <li class="social-icon">
+          <a href="" target="_blank"><i class="fab fa-twitter"></i></a>
+        </li>
+        <li class="social-icon">
+          <a href="" target="_blank"><i class="fab fa-linkedin-in"></i></a>
+        </li>
+        <li class="social-icon">
+          <a href="" target="_blank"> <i class="fab fa-medium-m"></i></a>
+        </li>
+        <li class="social-icon">
+          <a href="" target="_blank"><i class="fab fa-github"></i></a>
+        </li>
+        <li class="social-icon">
+          <a href="" target="_blank"><i class="fab fa-angellist"></i></a>
+        </li>
+      </ul>
+      <!-- Button -->
+      <div class="header-button">
+        <i class="fas fa-chevron-down"></i>
+      </div>
+      <!-- End button -->
+    </section>
+    <!-- End Headline -->
+    <!-- Works -->
+    <section class="section">
+      <div class="works-title-wrapper">
+        <h2>
+          My Recent <br />
+          Works
+        </h2>
+        <div class="underline"></div>
+      </div>
+      <!-- Projects -->
+      <div class="cards-container">
+        <!-- project -->
+        <div class="project">
+          <div class="image-wrapper"></div>
+          <div class="project-footer">
+            <div class="project-title-wrapper">
+              <h4>Multi-Post Stories<br />Gain+Glory</h4>
+            </div>
+            <ul class="project-buttons-wrapper">
+              <li>
+                <a href="#"
+                  ><button class="btn-hover btn-active btn-pressed">
+                    Ruby on rails
+                  </button></a
+                >
+              </li>
+              <li>
+                <a href="#"
+                  ><button class="btn-hover btn-active btn-pressed">
+                    CSS
+                  </button></a
+                >
+              </li>
+              <li>
+                <a href="#"
+                  ><button class="btn-hover btn-active btn-pressed">
+                    javascript
+                  </button></a
+                >
+              </li>
+              <li>
+                <a href="#"
+                  ><button class="btn-hover btn-active btn-pressed">
+                    HTML
+                  </button></a
+                >
+              </li>
+            </ul>
+            <button class="section-button btn-hover btn-active btn-pressed">
+              See Project
+            </button>
+          </div>
```

Comment:
- [Optional ] You did well implementing the changes required by the previous reviewer for the button styling: Active, Hover, and Pressed. Consider using the same colors as the Figma design.